variable.c

```diff
@@ -189,15 +189,25 @@ is_constant_path(VALUE name)
  *    m.set_temporary_name(nil) # => #<Module:0x0000000102c68f38>
  *    m.name #=> nil
  *
- *    n = Module.new
- *    n.set_temporary_name("fake_name")
+ *    c = Class.new do
```

Comment:
I'm with @mame here, the method already has significant examples, I don't think we need to add to them.

Reply:
OK. My point here was to demonstrate one of the cases where redefining `Module#name` and/or `Module#inspect` wouldn't fix the related object's `#inspect` to be meaningful (and adds to a justification "why somebody needs a new method to set a pretend-name"). But probably, it doesn't look important enough anyway.